src/SmartComponents/CompliancePolicies/CompliancePolicies.js

```diff
@@ -20,9 +19,11 @@ import useAPIV2FeatureFlag from '../../Utilities/hooks/useAPIV2FeatureFlag';
 import PropTypes from 'prop-types';
 import dataSerialiser from '../../Utilities/dataSerialiser';
 import { QUERY, dataMap } from './constants';
+import TableStateProvider from '../../Frameworks/AsyncTableTools/components/TableStateProvider';
```

Comment:
Let's use path alias for imports
```suggestion
import TableStateProvider from '@/Frameworks/AsyncTableTools/components/TableStateProvider';
```